servers/cromwell/README.md

```diff
@@ -38,15 +38,41 @@ Thin shim around [`cromwell`](https://github.com/broadinstitute/cromwell).
     {
       "field": "labels.cromwell-workflow-id",
       "display": "Workflow ID"
+    },
+    {
+      "field": "labels.flag",
+      "display": "Flag",
+      "editable": true,
+      "bulkEditable": true,
+      "fieldType": "list",
+      "validFieldValues": [
+        "archive",
+        "follow-up"
+      ]
+    },
+    {
+      "field": "labels.label",
+      "display": "Label",
+      "fieldType": "text",
+      "editable": true,
+      "bulkEditable": true
+    },
+    {
+      "field": "labels.comment",
+      "display": "Comment",
+      "fieldType": "text",
+      "editable": true
     }
   ],
   "commonLabels": [
     "cromwell-workflow-id",
-    "workflow-name"
+    "workflow-name",
```

Comment:
Same as below, I think we would include `label` and `comment` here so that these fields appear in the query builder